lib/dmv.rb

```diff
@@ -1,15 +1,17 @@
 class Dmv
 
+  attr_reader :facilities
   def initialize
     @facilities = []
   end
 
+
   def add_facility(facility)
-    @facilities << facility
+    @facilities.push(facility)
```

Comment:
You can use either the shovel ( << ) or .push here! They accomplish the same goal. The .push method can take multiple arguments but in this case (with 1 argument) there's no difference between what they can do